WooCommerce/WooCommerceTests/ViewRelated/Products/Edit Product/ProductFormViewModel+UpdatesTests.swift

```diff
@@ -225,6 +225,25 @@ final class ProductFormViewModel_UpdatesTests: XCTestCase {
         // Assert
         XCTAssertEqual(viewModel.product.sku, sku)
     }
+
+    func testUpdatingExternalLink() {
+        // Arrange
+        let product = MockProduct().product(sku: "")
```

Comment:
You probably just wanted: 

```suggestion
        let product = MockProduct().product()
```

😁

Reply:
cool! updated in https://github.com/woocommerce/woocommerce-ios/pull/2442/commits/bc3de89a077dd574de02c7cb19a2e9c71e17b368